app/src/main/java/org/coepi/android/ui/onboarding/binders/SmallOnboardingCardBinder.kt

```diff
@@ -0,0 +1,37 @@
+package org.coepi.android.ui.onboarding.binders
+
+import android.view.View
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.card_onboarding_small.view.*
+import org.coepi.android.R
+import org.coepi.android.ui.onboarding.OnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingCard.SmallOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingClickEvent
+
+object SmallOnboardingCardBinder: OnboardingCardBinder {
+
+    override fun bind(
+        itemView: View,
+        onboardingCard: OnboardingCard,
+        onboardingEventSubject: Subject<OnboardingClickEvent>
+    ) {
+        onboardingCard as SmallOnboardingCard
+
+        itemView.onboarding_card_small_title.text = onboardingCard.title
+        itemView.onboarding_card_small_content.text = onboardingCard.message
+
+        when (onboardingCard.order) {
+            0 -> itemView.progress_icon_1.setImageResource(R.drawable.stepper_icon_selected)
```

Comment:
Add the image resource to the view data in the view model. This is again too much logic for the view. View data should be "ready to consume".